azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatActionHandler.kt

```diff
@@ -120,15 +120,16 @@ internal class ChatActionHandler(private val chatService: ChatService) {
         chatService.editMessage(action.message.normalizedID.toString(), action.message.content ?: "")
             .whenComplete { _, error ->
                 if (error != null) {
-                    dispatch(
-                        ErrorAction.ChatStateErrorOccurred(
-                            chatCompositeErrorEvent = ChatCompositeErrorEvent(
-                                threadId,
-                                ChatCompositeErrorCode.SEND_EDIT_MESSAGE_FAILED,
-                                error
-                            )
-                        )
-                    )
+                    // TODO will add back when we release the edit logic
```

Comment:
I would not comment that out. Even if we do not have this feature available on UI, we still have this method (and many more) that related to the Edit feature. It does not make much sense to me to comment out only an error handling part of it.

Reply:
The reason is we don't want to expose the ChatCompositeErrorCode.SEND_EDIT_MESSAGE_FAILED, because we cannot edit the message for now.

So I just comment this error handle logic.